app/src/api.js

```diff
@@ -128,3 +135,33 @@ export const statuses = {
   stale: "stale", // results superseded by newer query
   // else: error fetching results
 };
+
+// get external link for tag metadata
+const getTagLink = (tag) => {
+  if (!tag) return "";
+
+  // get parts of tag
+  const prefix = tag.split("_").shift().toLowerCase();
+  const id = tag.split("_").pop().toUpperCase();
+
+  // map of links
+  const links = {
+    gene: "https://www.ncbi.nlm.nih.gov/gene/<id>",
+    species:
+      "https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=info&id=<id>",
+    cellline: "https://web.expasy.org/cellosaurus/<id>",
+    disease: "https://id.nlm.nih.gov/mesh/<id>",
+    chemical: "https://id.nlm.nih.gov/mesh/<id>",
+  };
+
+  // find matching link
+  const link = find(links, (value, key) => prefix === key)?.replace("<id>", id);
+
+  // https://fontawesome.com/docs/apis/javascript/methods
+  const extIcon = icon(faExternalLinkAlt, {
+    styles: { "margin-left": "0.5em" },
+  }).html;
+
+  // return html to show
+  return link ? `<a href="${link}">Metadata${extIcon}</a>` : "Unknown tag type";
```

Comment:
I'd personally add a `target="_blank"` attribute here so it opens a new window versus navigating away from the current view. Also, I wonder if it'd save some space and be a bit more semantic to have the name of the tag be the link text rather than "Metadata".